src/test/unit_tests/test_public_access_block.js

```diff
@@ -110,29 +110,33 @@ mocha.describe('noobaa public access block test', function() {
 
     mocha.it('put_public_access_block must throw when acls are used', async function() {
         await run_on_random_bucket(s3_owner, bucket_prefix, async bucket => {
-            try {
```

Comment:
what is the difference between the old tests and the new? can you add a short explanation in the PR of why did the test not work, and what you changed

Reply:
Right, actually `assert.fail` throws but the finally runs anyway so basically its like `.catch(_.noop)`. Which gives us false positives.